tests/c/awkward_unmappable.c

```diff
@@ -0,0 +1,26 @@
+// ignore: Fails with: Assertion `!CallStack.curMappableFrame()' failed
+// Run-time:
+//   env-var: YKD_SERIALISE_COMPILATION=1
+//   status: success
+
+#include <assert.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
```

Comment:
I think the headers could be minimised. string and stdio don't appear to be used?

Reply:
Good point. Fixed in 0143a1a.